bin/dump_restbase

```diff
@@ -15,18 +18,43 @@ var argParser = require('yargs')
         describe: 'Verbose logging',
     })
     .options('workDir', {
-        default: '/tmp',
         describe: 'Directory to use for in-progress dump files',
     })
     .options('dumpDir', {
-        default: '/tmp',
         describe: 'Directory to use for finished dump files',
     });
-var options = argParser.argv;
+var argv = argParser.argv;
 
-if (options.h) {
+if (argv.h) {
     argParser.showHelp();
     process.exit(1);
 }
 
-return dumpAllWikis(options);
+// Default global options
+var options = { maxConcurrency : 50, concurrency: 3 };
```

Comment:
These default option are consumed when config file is not specified and command line options are used